.gitignore

```diff
@@ -5,3 +5,5 @@ dist/
 build/
 dLuxToliman.egg-info/
 .idea/
+.vscode/
+.gitignore
```

Comment:
Is this so everyone can maintain their own gitignore?

Reply:
ye

Reply:
No worries